addons/web_editor/static/lib/odoo-editor/demo/main.js

```diff
@@ -0,0 +1,96 @@
+import { OdooEditor } from '../src/OdooEditor.js';
+
+const localStorageKey = 'odoo-editor-localHtmlSaved';
+
+function startEditor(testHTML) {
+    const editableContainer = document.getElementById('dom');
+    editableContainer.innerHTML = testHTML;
```

Comment:
Security hazard right here ?

Reply:
That's right, I'm removing feature that allow us to pass arbitrary html through the url.

Reply:
I'm not using innerHTML but with content that was saved to the local storage in a relatively safe manner.